vector/src/main/res/layout/dialog_change_password.xml

```diff
@@ -1,31 +1,67 @@
+    tools:targetApi="lollipop_mr1">
 
-    <EditText
-        android:id="@+id/change_password_old_pwd_text"
+    <android.support.design.widget.TextInputLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:hint="@string/settings_old_password"
-        android:inputType="textPassword" />
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/change_password_old_pwd_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/settings_old_password"
+            android:inputType="textPassword"
+            android:padding="16dp" />
+    </android.support.design.widget.TextInputLayout>
 
-    <EditText
-        android:id="@+id/change_password_new_pwd_text"
+    <android.support.design.widget.TextInputLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:hint="@string/settings_new_password"
-        android:inputType="textPassword" />
+        android:layout_height="wrap_content">
 
-    <EditText
-        android:id="@+id/change_password_confirm_new_pwd_text"
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/change_password_new_pwd_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/settings_new_password"
+            android:inputType="textPassword"
+            android:padding="16dp" />
+    </android.support.design.widget.TextInputLayout>
+
+    <android.support.design.widget.TextInputLayout
         android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/change_password_confirm_new_pwd_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/settings_confirm_password"
+            android:inputType="textPassword"
+            android:paddingStart="16dp"
+            android:paddingLeft="16dp"
+            android:paddingTop="16dp"
+            android:paddingEnd="8dp"
+            android:paddingRight="8dp"
+            android:paddingBottom="16dp" />
+    </android.support.design.widget.TextInputLayout>
+
+    <TextView
+        android:id="@+id/pass_match_label"
+        android:layout_gravity="center_horizontal"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:hint="@string/settings_confirm_password"
-        android:inputType="textPassword" />
+        android:gravity="center"
+        android:padding="16dp"
+        tools:textColor="@android:color/holo_red_dark"
+        tools:drawableRight="@drawable/ic_alert_mark"
+        tools:text="   Passwords do not match    "/>
```

Comment:
you can use @string attributes

Reply:
and android:drawablePadding

Reply:
I meant to use ` android:drawablePadding` . Forgot to do it in hurry